### PR TITLE
Provide Indonesian wikilingua prompt

### PR DESCRIPTION
**Pull request by afaji**
Indonesian translation for wikilingua 
cc: @haileyschoelkopf @gentaiscool

### CODE REVIEW

**Review by gentaiscool — APPROVED** [review]
the translation looks good to me. I will approve this PR. I think we need an approval from a reviewer with write access.

**Review by VictorSanh — APPROVED** [review]
Looks good to me, i trust @gentaiscool who verified the Indonesian part, merging now

**Pull request merged**
